packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,89 @@ section: get-started
+
+### Component groups restructuring
+
+We made many updates to our component groups extension, to improve accuracy, usability, and alignment with PatternFly 6. We've moved its website section to our top-level navigation for better visibility, and also to support necessary sub-navigation. We've conceptually grouped the components into functional categories. Additionally, we renamed some of the components to be more accurate. 
+
+[Check out the updated documentation](/component-groups/about-component-groups), which includes:
+
+| **Category** | **Components** | 
+| --- | --- |
+| Content containers | - Details page <br /> - Multi-content card<br /> - Page header (previously named "content header")<br /> - Service card (new!) |
+| Controls | - Bulk select <br /> - Close button <br /> - Responsive actions (new!) |
+| Error communication | - Error boundary <br /> - Error state <br /> - Missing page (previously named "invalid object") <br /> - Unauthorized access (previously named "not authorized") <br /> - Unavailable content <br /> - Warning modal |
+| Helpers | - Column management modal <br /> - Log snippet <br /> - Shortcut grid |
+| Status and state indicators | - Ansible <br /> - Severity (previously Battery) <br /> - Skeleton table <br /> - Status (new!) <br /> - Tag count |
+
+### React component updates 
+
+Here are the most significant updates we made to our React components: 
+
+- Data list
+    - Refactored `<DataListCheck>` to use `<Checkbox>` internally. As a result, `id` is now a required prop for ` <DataListCheck>`. This also fixes broken checkboxes in the examples. 
+- Dropdown
+    - Added delay before setting focus on the first dropdown item when `shouldFocusFirstItemOnOpen` is set. This gives enough time for the popover to be shown in the correct location, so the scrollable doesn't scroll to an invalid location.
+- Jumplinks
+    - Fixed improper offset in [demo](/components/jump-links/react-demos). 
+- Menu toggle
+    - Removed `pf-m-actions` and, consequently, `SplitButtonOptions`. Items should now be passed directly to `splitButtonItems`.
+    - Added `isPlaceholder` to support customizable placeholder text, as well as [a corresponding example](/components/menu-toggle#placeholder-text-in-toggle).
+
+### Token updates 
+
+#### Design tokens 
+
+We updated existing design tokens and added new tokens to support directional box-shadows. To see these tokens, search "box-tokens" in our [tokens documentation](/tokens/all-patternfly-tokens).
+
+We also added a few new tokens to support the needs of our new chatbot, including:
+- A tertiary background token that accommodates containers placed on a secondary background.
+- Inverse hover and inverse clicked tokens.
+
+#### React tokens
+
+To address instances where chart tokens and chart variable names were unintentionally identical, we added a `t_` prefix to our React tokens. This makes it easier to differentiate between tokens and variables.
```

Comment:
Maybe say `between design tokens and other CSS variables.` or whatever that actually is, but it's in a React tokens section and referring to design token variables.

Reply:
You got it! That update lgtm.

> "we added a t_ prefix to all design tokens in React tokens"
> the prefix wasn't added to the React tokens themselves? I thought this part was the React token, am I misunderstanding?

Yep it's on the react token. When I sad "all design tokens in react tokens" I'm thinking of it like react tokens are a thing themselves, and are made up of design tokens and CSS vars. So with _design token in react tokens_, I'm just referring to any of the `t_` react tokens.